### DISCUSSION TITLE
处理器感知线程管理系统 · OSDI 2018 - 面向信仰编程 · /papers-arachne

### DISCUSSION THREAD

**Discussion by draveness** [draveness/blog-comments#620] [Announcements]
https://draveness.me/papers-arachne/

**Comment by Zheaoli** [comment]
你这读 paper 的效率也贼高了点，，这篇论文的思路很有趣，不过挺好奇 Arachne 这种旁路做法怎么去保证 Arachne 的可用性（捂嘴

**Comment by draveness** [comment]
> 你这读 paper 的效率也贼高了点，，这篇论文的思路很有趣

也没有吧，看了一段时间

> 不过挺好奇 Arachne 这种旁路做法怎么去保证 Arachne 的可用性（捂嘴

个人觉得还好，主要是这种设计思路

**Comment by NCZkevin** [comment]
博主现在做基础架构吗？难得看见一篇这种解析论文的文章

**Comment by wujunwei** [comment]
博主有空能讲讲google的三驾马车吗？

**Comment by successli** [comment]
一般博客系统是由hexo jekyll等生成，请问博主的博客网站是用了哪个框架，是否开源了？

**Comment by draveness** [comment]
> 一般博客系统是由hexo jekyll等生成，请问博主的博客网站是用了哪个框架，是否开源了？

hugo，没有

**Comment by successli** [comment]
希望博主可以聊一聊博客页面的构想和搭建流程～

**Comment by hudingyu** [comment]
博客停更俩月了，有点反常啊

**Comment by draveness** [comment]
> 博客停更俩月了，有点反常啊

最近事情比较多
